pkg/agentconfig/container.go

```diff
@@ -216,3 +221,12 @@ func appendAppContainerEnvFrom(app *core.Container, cc *Container, es []core.Env
 	}
 	return es
 }
+
+func getIgnoredVolumeMounts(annotations map[string]string) map[string]bool {
+	vmSlice := strings.Split(annotations["telepresence.getambassador.io/inject-ignore-volume-mounts"], ",")
+	ivms := make(map[string]bool, len(vmSlice))
```

Comment:
A bit of a nit-pick, and I'm fine if you leave it as is, but the usage for `ivms` is a unique set of strings. This gives the impression that it's a map of `true` or `false` values. I usually use `map[string]struct{}` (and `if _, ok = ignoredVolumeMounts[m_name]; ok {` on line 194) to make sure it's understood that the value is insignificant.